c/tskit/trees.c

```diff
@@ -4624,6 +4624,7 @@ tsk_treeseq_genetic_relatedness(const tsk_treeseq_t *self, tsk_size_t num_sample
     const double *windows, tsk_flags_t options, double *result)
 {
     int ret = 0;
+    // we want to allow self comparisons
```

Comment:
maybe expand this comment to explain the `2` below? also, maybe we want the comment to be C-style rather than C++, i.e. `/* */

Reply:
oh whoops, this is outdated, since the `2` is the tuple size; there's nothing in this code now that explicitly allows self comparisons. I'll remove it.
```suggestion
```